api/v1beta1/spiaccesscheck_types.go

```diff
@@ -42,7 +42,8 @@ type SPIAccessCheckStatus struct {
 type SPIRepoType string
 
 const (
-	SPIRepoTypeGit SPIRepoType = "git"
+	SPIRepoTypeGit       SPIRepoType = "git"
+	SPIRepoTypeContainer SPIRepoType = "container"
```

Comment:
Would not something like "image" or "dockerImage" look more precise?

Reply:
`image` is imo too general and might sound like picture image and I have issue with word `docker` because it's just one project that works with containers (even though it's very standardised word).

I think best describing would be `containerImageRegistry`, but it's quite long. Or maybe `containerRegistry`. I've used `container` here because I think it's short and clear.

We must also consider that target of this API is some other component.

cc: @skabashnyuk @mshaposhnik @metlos @xbaran4 opinions?